### ISSUE TITLE
Specify base_uri as an environment variable

### ISSUE THREAD

**Issue by JoyceBabu** [mobz/elasticsearch-head#332]
Is it possible to specify `base_uri` as an environment variable, when using docker?

**Comment by marclennox** [comment]
+1 for this.  Alternatively, being able to run in proxy mode by supplying the proxy json configuration as a volume mapping would work.

**Comment by mobz** [comment]
@marclennox This sounds like an interested posibility
The way proxy will work is by creating a proxy to each cluster defined in the clusters folder,
so you could allow the path to the clusters folder to be an env variable and then docker can provide a mapped directory at that point? Is this what you are thinking?

**Comment by marclennox** [comment]
@mobz yes exactly what I was thinking!  I'm assuming that in proxy mode, your browser doesn't need to open a connection directly to elasticsearch?

**Comment by mobz** [comment]
that's correct. `recipes` in proxy mode provide a way to traverse complex network topologies (eg burrow through a firewall, or access some cloud provider), while `clusters` provide configuration for specific instances of clusters
